test-cases/scylla-operator/kubernetes-platform-upgrade-gke.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 660
+
+prepare_write_cmd: [
+    # 40Gb
+    "cassandra-stress write no-warmup cl=ALL n=10000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..10000000",
+    "cassandra-stress write no-warmup cl=ALL n=10000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..20000000",
+    "cassandra-stress write no-warmup cl=ALL n=10000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=20000001..30000000",
+    "cassandra-stress write no-warmup cl=ALL n=10000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=30000001..400000000"
+]
+
+stress_during_entire_upgrade: "cassandra-stress read no-warmup cl=QUORUM duration=300m -schema 'replication(factor=3)' -mode cql3 native -rate 'threads=100 throttle=5000/s' -col 'size=FIXED(1024) n=FIXED(1)' -pop 'dist=gauss(1..60000000,30000000,15000000)' -log interval=5"
```

Comment:
this is currently the only diff between the two configurations, wouldn't it be enough to leave `kubernetes-platform-upgrade.yaml` and create configuration only with what's actually different ?

Reply:
Ok, will change